html/Timer.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<!--    fourth take-->
+
+    <meta charset="UTF-8">
+    <title>Timer</title>
+    <link rel="stylesheet" href="Timer.css">
```

Comment:
When I clone your code, it creates 3 directories:
- Javascript
- css
- html
In order to run your application, I had to copy the css, html, and js file in once directory and then run it.
Correct it either by moving them all in one directory or use relative addressing to load them from the right place (the latter is preferred).

Reply:
Sorry about that! I fixed the relative path. I made the  separate directories after writing my code, and forgot to modify the relative paths.